src/Install/Installer.php

```diff
@@ -78,96 +78,9 @@ public function install()
             }
         }
 
-        if (!$this->enableDefaultPaymentSettings()) {
-            $this->errors[] = $this->module->l('Failed to enable default payment settings', __CLASS__);
-            return false;
-        }
-
         return true;
     }
 
-    /**
-     * @return bool
-     */
-    private function enableDefaultPaymentSettings()
-    {
-        /** @var SaferPayPaymentCreator $paymentCreator */
-        $paymentCreator = $this->module->getModuleContainer()->get(SaferPayPaymentCreator::class);
-
-        /** @var SaferPayLogoCreator $logoCreator */
-        $logoCreator = $this->module->getModuleContainer()->get(SaferPayLogoCreator::class);
-
-        /** @var SaferPayRestrictionCreator $restrictionCreator */
-        $restrictionCreator = $this->module->getModuleContainer()->get(SaferPayRestrictionCreator::class);
-
-        $success = true;
-
-        foreach (SaferPayConfig::PAYMENT_METHODS as $paymentMethod) {
-            $success &= $this->enablePayment($paymentMethod, $paymentCreator);
-            $success &= $this->enableLogo($paymentMethod, $logoCreator);
```

Comment:
You are checking if logo is enabled for displaying payment options but wouldn't deletion of this installer make that everything was disabled on installation?

Reply:
Yes, in this case merchant must connect to saferpay account first and only after that he will be able enable payments 😅